go/libkb/pgp_key.go

```diff
@@ -607,6 +609,13 @@ func (k *PGPKeyBundle) CheckFingerprint(fp *PGPFingerprint) error {
 }
 
 func (k *PGPKeyBundle) SignToString(msg []byte) (sig string, id keybase1.SigID, err error) {
+	if k.Entity.PrivateKey == nil || k.Entity.PrivateKey.PrivateKey == nil {
```

Comment:
Interesting approach.  A few thoughts:
* do we always want this behavior throughout our code?
* what happens on installs without gpg access?
* will this be a performance hit (to be shelling out to GPG when it might just fail everytime?)